api/repositories/app_repository_test.go

```diff
@@ -53,8 +53,8 @@ var _ = Describe("AppRepository", func() {
 		space2 = createSpaceAnchorAndNamespace(testCtx, org.Name, prefixedGUID("space2"))
 		space3 = createSpaceAnchorAndNamespace(testCtx, org.Name, prefixedGUID("space3"))
 
-		spaceDeveloperClusterRole = createSpaceDeveloperClusterRole(testCtx)
-		spaceAuditorClusterRole = createSpaceAuditorClusterRole(testCtx)
+		spaceDeveloperClusterRole = createClusterRole(testCtx, repositories.SpaceDeveloperClusterRoleRules)
```

Comment:
Provided that `spaceDeveloperClusterRole` and `spaceAuditorClusterRole` are clusterwide resources, they would not be deleted automatically on namespace destruction and I guess they would leak. How about deleting them on `AfterEach`?

This probably applies to all roles created via `createClusterRole`

Reply:
As discussed over slack, these are env tests so clusterwide resoources are destroyed anyway. So no need to cleanup explicitly.